infra-as-code/bicep/modules/vwanConnectivity/vwanConnectivity.bicep

```diff
@@ -20,32 +17,53 @@ param parAzFirewallTier string = 'Standard'
 @sys.description('Switch to enable/disable Virtual Hub deployment.')
 param parVirtualHubEnabled bool = true
 
-@sys.description('Switch to enable/disable VPN Gateway deployment.')
-param parVpnGatewayEnabled bool = true
-
-@sys.description('Switch to enable/disable ExpressRoute Gateway deployment.')
-param parExpressRouteGatewayEnabled bool = true
-
-@sys.description('Switch to enable/disable Azure Firewall deployment.')
-param parAzFirewallEnabled bool = true
-
 @sys.description('Switch to enable/disable Azure Firewall DNS Proxy.')
 param parAzFirewallDnsProxyEnabled bool = true
 
 @sys.description('Prefix Used for Virtual WAN.')
 param parVirtualWanName string = '${parCompanyPrefix}-vwan-${parLocation}'
 
 @sys.description('Prefix Used for Virtual WAN Hub.')
-param parVirtualWanHubName string = '${parCompanyPrefix}-vhub-${parLocation}'
+param parVirtualWanHubName string = '${parCompanyPrefix}-vhub'
+
+@sys.description('''
+Array Used for multiple Virtual WAN Hubs deployment. Each object in the array represents an individual Virtual WAN Hub configuration. Add/remove additional objects to meet the number of Virtual WAN Hub requirement.
+"parVpnGatewayEnabled" - Switch to enable/disable VPN Gateway deployment on the respective Virtual WAN Hub.
+"parExpressRouteGatewayEnabled" - Switch to enable/disable ExpressRoute Gateway deployment on the respective Virtual WAN Hub.
+"parAzFirewallEnabled" - Switch to enable/disable Azure Firewall deployment on the respective Virtual WAN Hub.
+"parVirtualHubAddressPrefix" - The IP address range in CIDR notation for the vWAN virtual Hub to use.
+"parHublocation" - The Virtual WAN Hub location.
+"parHubRoutingPreference" - The Virtual WAN Hub routing preference. The allowed values are ASN, VpnGateway, ExpressRoute.
+"parVirtualRouterAutoScaleConfiguration" - The Virtual WAN Hub capacity. The value should be between 2 to 50.
+''')
+param parVirtualWanHubs array = [ {
+    parVpnGatewayEnabled: true
+    parExpressRouteGatewayEnabled: true
+    parAzFirewallEnabled: true
+    parVirtualHubAddressPrefix: '10.100.0.0/23'
+    parHublocation: 'centralus'
+    parHubRoutingPreference:'' //allowed values are 'ASN','VpnGateway','ExpressRoute'
```

Comment:
Yes @jtracey93 , the omitted parameters take "ExpressRoute" as the default value. I will add the description that mentions omitting this will add "ExpressRoute" as default.